recipes/onnxruntime/all/conanfile.py

```diff
@@ -0,0 +1,234 @@
+        tc.variables["onnxruntime_BUILD_BENCHMARKS"] = False
+        tc.variables["onnxruntime_USE_ROCM"] = False
+        tc.variables["onnxruntime_GCOV_COVERAGE"] = False
+        tc.variables["onnxruntime_USE_MPI"] = False
+        tc.variables["onnxruntime_ENABLE_MEMORY_PROFILE"] = False
+        tc.variables["onnxruntime_ENABLE_CUDA_LINE_NUMBER_INFO"] = False
+        tc.variables["onnxruntime_BUILD_WEBASSEMBLY"] = False
+        tc.variables["onnxruntime_BUILD_WEBASSEMBLY_STATIC_LIB"] = False
+        tc.variables["onnxruntime_ENABLE_WEBASSEMBLY_EXCEPTION_CATCHING"] = False
+        tc.variables["onnxruntime_ENABLE_WEBASSEMBLY_API_EXCEPTION_CATCHING"] = False
+        tc.variables["onnxruntime_ENABLE_WEBASSEMBLY_EXCEPTION_THROWING"] = False
+        tc.variables["onnxruntime_ENABLE_WEBASSEMBLY_THREADS"] = False
+        tc.variables["onnxruntime_ENABLE_WEBASSEMBLY_DEBUG_INFO"] = False
+        tc.variables["onnxruntime_ENABLE_WEBASSEMBLY_PROFILING"] = False
+        tc.variables["onnxruntime_ENABLE_EAGER_MODE"] = False
+        tc.variables["onnxruntime_ENABLE_LAZY_TENSOR"] = False
+        tc.variables["onnxruntime_ENABLE_EXTERNAL_CUSTOM_OP_SCHEMAS"] = False
+        tc.variables["onnxruntime_ENABLE_CUDA_PROFILING"] = False
+        tc.variables["onnxruntime_ENABLE_ROCM_PROFILING"] = False
+        tc.variables["onnxruntime_USE_XNNPACK"] = False
+        tc.variables["onnxruntime_USE_CANN"] = False
+        tc.generate()
+        deps = CMakeDeps(self)
+        deps.generate()
+        vbe = VirtualBuildEnv(self)
+        vbe.generate(scope="build")
+
+    def build(self):
+        apply_conandata_patches(self)
+        cmake = CMake(self)
+        cmake.configure(build_script_folder="cmake", cli_args=["--compile-no-warning-as-error"])
+        cmake.build()
+
+    def package(self):
+        copy(self, pattern="LICENSE", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        cmake = CMake(self)
+        cmake.install()
+        pkg_config_dir = os.path.join(self.package_folder, "lib", "pkgconfig")
+        rmdir(self, pkg_config_dir)
+
+    def package_info(self):
+        if self.options.shared:
+            self.cpp_info.libs = ["onnxruntime"]
+        else:
+            onnxruntime_libs = [
+                "session",
+                "optimizer",
+                "providers",
+                "framework",
+                "graph",
+                "util",
+                "mlas",
+                "common",
+                "flatbuffers",
+            ]
+            self.cpp_info.libs = [f"onnxruntime_{lib}" for lib in onnxruntime_libs]
+
+        self.cpp_info.includedirs.append("include/onnxruntime/core/session")
+
+        if self.settings.os in ["Linux", "Android", "FreeBSD", "SunOS", "AIX"]:
+            self.cpp_info.system_libs.append("m")
+            self.cpp_info.system_libs.append("pthread")
+        if is_apple_os(self):
+            self.cpp_info.frameworks.append("Foundation")
```

Comment:
```suggestion
            self.cpp_info.frameworks.append("Foundation")
            
        self.cpp_info.set_property("pkg_config_name", "onnxruntime")
  
```
onnxruntime seems to generate a PkgConfig file
https://github.com/microsoft/onnxruntime/blob/main/cmake/libonnxruntime.pc.cmake.in